README.md

```diff
@@ -22,16 +22,27 @@ Checkout our [wiki](https://github.com/gnosis/dex-contracts/wiki/Script-Usage-Ex
 
 For the deployment of the contracts into an official network, follow this steps:
 
-1. Make sure that all depended contracts and libraries - e.g. BytesLib - has been deployed to the intended network and that their network information is available in the npm modules 
+1. Make sure that all depended contracts and libraries - e.g. BytesLib - has been deployed to the intended network and that their network information is available in the npm modules
 
 2. Run the following commands
-```
+```sh
 npm install                         // This installs all dependencies
 npx truffle build                   // This builds the contracts
-npx truffle migrate --network NETWORKNAME --reset
+npx truffle migrate --network $NETWORKNAME --reset
 npm run networks-extract            // extracts deployed addresses to networks.json
 ```
 
+3. Verify the contracts for some cool Etherscan.io goodies
+```sh
+npx truffle run verify SnappAuction --network $NETWORKNAME
+npx truffle run verify StablecoinConverter --network $NETWORKNAME
+```
```

Comment:
Could we just add this note:

Note, if you have a specific contract address in mind (i.e. one which is not specified in `networks.json`) it may be referred to by address as

```
truffle run verify <ContractName>@<CONTRACT_ADDRESS> --network rinkeby
```